### ISSUE TITLE
Include option to disable overscan area for NES

### ISSUE THREAD

**Issue by msikma** [jasarien/Provenance#436]
A lot of NES emulators provide an option to cut off an overscan area to simulate the slightly smaller viewable area of the NES image on a TV. Although [overscan](https://wiki.nesdev.com/w/index.php/Overscan) is technically complicated and partly TV-dependent, emulators generally cut the picture from 256x240 to 256x224. This is because some games have glitch pixels in the far top and bottom of the screen since they did not get displayed on a TV anyway.

It would be nice to have an option in the settings to cut off the top and bottom 8 pixels, called something like "Trim NES border area."

(This might also be useful for other consoles, I'm not sure.)

I'm using a build from `master`, commit `a3e072b`.

**Comment by msikma** [comment]
Is there any chance that I could get an invite for the dev Slack? I've been trying to work on this, but I'm looking for the best place to actually take off the pixels. Some guidance would be ideal. Right now I'm looking at `Provenance/Emulator/PVGLViewController.m/glkView:drawInRect` but I'm not sure that's the best place.

**Comment by jasarien** [comment]
Let me know an email for the invite. As far as the pixel trimming, it's more likely to be an option in the emulator itself than something we want to do manually on rendering.

**Comment by msikma** [comment]
Yeah, you're right. I'm thinking that it's a setting (to enable/disable) which gets looked up and utilized when the emulator core returns its rendered buffer. If you could send an invite to mike@letsdeliver.com I would appreciate it :)

**Comment by sevdestruct** [comment]
There has been discussion about this, I need to pull it here, as I think there is differences across cores so we might need something like a multi frame check to find out what the dead space is before it gets dropped (check the bottom of Chrono trigger for instance, or final fantasy vii, the sides of Mario on new or all NES games which Joe found some overscsn documentation about soecifixslly. 

Ah, the notes are in my Trello [board](https://trello.com/c/p94MwtOX/23-black-color-dead-space-buffering-bar-auto-cropping).
